packages/web-components/src/components/ic-status-tag/ic-status-tag.tsx

```diff
@@ -71,7 +71,9 @@ export class StatusTag {
             [`${appearance}-${status}`]: appearance !== undefined,
             [`${variant}-${status}`]: true,
             ["outlined"]: variant === "outlined",
-            ["small"]: small || size === "small",
```

Comment:
Some, if not all of this line should still be in the code. Using the deprecated prop: `small` should still have an effect on the size of the status tag for the time being. This prop will be removed in V3